docs/build-on-linea/tooling/cross-chain/shortcuts.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Onthis Shortcuts
+description: No-code tool for executing multiple DeFi actions in one transaction
+---
+
+A Shortcut allows users to execute onchain actions by sending funds to a contract address.
+
+Each Shortcut is a unique smart contract designed to carry out the desired function. Shortcuts are
+represented by ENS domains, making addresses human-readable.
+
+:::note
+
+Shortcuts currently works with Ethereum Mainnet only, unless explicitly specified by the Shortcut 
+description. Shortcut addresses are not chain-agnostic, but are different from network to network. 
+Sending ETH (or other tokens) to the wrong network may result in a loss of funds.
+
+:::
+
+## How it works (user flow)
+
+A simple use case for a Shortcut could involve a user who wants to streamline the process of 
```

Comment:
Maybe this is the answer to my question: This works for Ethereum --> Linea flows.